### PR TITLE
feat(tests-unit): adds context/services/storage test

### PR DESCRIPTION
**Pull request by jurajpiar**
Includes contexts: global, offers, agreements

As part of the https://rsklabs.atlassian.net/browse/RMKT-457

### CODE REVIEW

**Review by itofarina — COMMENTED** [review]
Nice job! Just left a couple of comments for small improvements

**Review by 0xartem — COMMENTED** [review]
Looks good to me. As far as I understand these are unit-tests only. Are there any integration tests in this sprint?

File: src/context/Services/storage/__tests__/Context.tests.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+// eslint-disable-next-line import/no-extraneous-dependencies
```
[the remark this answers is not in the document]
Comment (jurajpiar) [reply src/context/Services/storage/__tests__/Context.tests.tsx:2]: Aah ..missed the type in the name. thanks

**Review by jurajpiar — COMMENTED** [review]
> Looks good to me. As far as I understand these are unit-tests only. Are there any integration tests in this sprint?

Don't think so. Let's discuss tho.

**Review by itofarina — APPROVED** [review]
LGTM!

**Pull request merged**